react-example/src/Components/LayoutLogin/LayoutLogin.less

```diff
@@ -0,0 +1,46 @@
+main{
+  display: grid;
+  width: 100vw;
+  grid-template-columns: 1fr 1fr;
+  height: 100vh;
+}
+
+.layout-login{
+  display: grid;
+  width: 100%;
+  background-color: #fff;
+  justify-content: center;
+  .div-login{
```

Comment:
Do a line break between rules and indented selectors.

✗ Instead of:

```
.layout-login{
  ...
  justify-content: center;
  .div-login{
```

✓ Do:

```
.layout-login{
  ...
  justify-content: center;

  .div-login{
```

Also, please fix this on all of the less files :)